waffle/models.py

```diff
@@ -481,5 +508,23 @@ class Meta:
 
     def is_active(self):
         if not self.pk:
+            if get_setting('CREATE_MISSING_SAMPLE'):
+                default_percent = 0.0
+
+                if get_setting('SAMPLE_DEFAULT'):
+                    default_percent = 100.0
```

Comment:
You want to use the setting value. This can be combined with the line above: `get_setting('SAMPLE_DEFAULT', 0)`.

Reply:
I wasn't trying to just get a default value, or at least I don't think I was :)

In the defaults `SAMPLE_DEFAULT` is a boolean value, however there isn't a boolean option for a Sample, it's expressed as a percentage, so I'm saying if `SAMPLE_DEFAULT` is true then that implies that you want the sample value to be 100, if I just grabbed the default value I'm testing if it's defined.

I've never really used Samples in production so I may be misunderstanding their intended use.

Reply:
Got it. I've never used samples, either.

Reply:
Got it. I've never used samples, either.